packages/workflow/src/Extensions/DateExtensions.ts

```diff
@@ -161,7 +162,11 @@ function isWeekend(date: Date): boolean {
 	return [DAYS.saturday, DAYS.sunday].includes(DateTime.fromJSDate(date).weekday);
 }
 
-function minus(date: Date | DateTime, extraArgs: unknown[]): Date {
+function minus(date: Date | DateTime, extraArgs: unknown[]): Date | DateTime {
```

Comment:
In order to remain consistent with the rest of the UI, shouldn't the functions always return primitives such as strings?